### PR TITLE
 update Tag according to the design system

### PR DESCRIPTION
**Pull request by l3satwik**

### CODE REVIEW

File: src/components/styles/Tag.styles.ts [hunk 1]
```diff
@@ -19,8 +18,13 @@ export const tagStyle = css({
     fontSize: 7,
     marginLeft: 5,
     height: "inherit",
-    padding: "9px 10px",
+    padding: "10px 10px",
     cursor: "pointer",
-    position: "relative"
+    position: "relative",
+    ":hover": {
```
Comment (ritz078) [inline src/components/styles/Tag.styles.ts]: is this working ?

File: src/theme/typography.ts [hunk 2]
```diff
@@ -84,6 +84,11 @@ export const typography: Typography = {
       color: colors.gray.darker,
       fontSize: fontSize.s
     },
+    bold: {
```
Comment (ritz078) [inline src/theme/typography.ts]: is this present in the design system typography file?

File: src/components/styles/Tag.styles.ts (hunk 1 above)
Comment (l3satwik) [reply src/components/styles/Tag.styles.ts]: yes

File: src/theme/typography.ts (hunk 2 above)
Comment (l3satwik) [reply src/theme/typography.ts]: Got it added to Typography.

File: src/components/__tests__/__snapshots__/tag.test.tsx.snap [hunk 3]
```diff
@@ -95,11 +99,17 @@ exports[`Component: Tag snapshot 2`] = `
   font-size: 7px;
   margin-left: 5px;
   height: inherit;
-  padding: 9px 10px;
+  padding: 10px 10px;
   cursor: pointer;
   position: relative;
 }
 
+.emotion-0 i:hover {
```
Comment (azizhk) [inline src/components/__tests__/__snapshots__/tag.test.tsx.snap]: Why are we targetting all `<i>`'s inside this class.
I know that we use `<i>` for icon but someone can create `<i>` in label prop which would also be targetted by this selector.

can we change to 
Tag.styles.ts
```js
export const iconClass = css({
  // icon styles
  ":hover": {
    // icon hover styles
  }
})
```

pebble/src/components/Tag.tsx
```jsx
import { tagStyle, iconClass } from "./styles/Tag.styles";

// in the component
<i className={cx('pi pi-icon-name', iconClass)} />
```

**Comment by ritz078** [comment]
@l3satwik I am merging this for now but make sure this is added in design system.

**Pull request merged**